packages/devtools_app/lib/src/screens/debugger/evaluate.dart

```diff
@@ -32,11 +32,11 @@ class ExpressionEvalField extends StatefulWidget {
 
 class _ExpressionEvalFieldState extends State<ExpressionEvalField>
     with AutoDisposeMixin, SearchFieldMixin {
-  AutoCompleteController _autoCompleteController;
+  late AutoCompleteController _autoCompleteController;
   int historyPosition = -1;
 
   String _activeWord = '';
-  List<String> _matches = [];
+  List<String?> _matches = [];
```

Comment:
do we want to allow null matches? probably not if we are null asserting below on line 117

Reply:
Nope, done!